src/jit-format/jit-format.cs

```diff
@@ -115,9 +118,57 @@ private void SetPlatform()
                 }
             }
 
+            private string GetRepoRoot()
```

Comment:
I'd prefer to just have a shared utils.cs file statically linked in. But yes, there is too much code duplication now.